corona-warn-app/cwa-app-android
# PDF export dialog and info screen (EXPOSUREAPP-9187)

Dialog and info screen for certificate export to PDF.

<img width="201" alt="Screenshot 2021-08-26 at 18 42 18" src="https://user-images.githubusercontent.com/64849422/130993635-b38498d8-d3a4-427a-b36d-ec25cd7bfcd2.png">

How to test:
1) Scan Vaccination, Test and Recovery certificates.
2) In `VaccinationDetailsViewModel`, `TestCertificateDetailsViewModel` and `RecoveryCertificateDetailsViewModel` check `onExport` function.
3) Change `exportPossible` to `false` to show dialog, to `true` to continue navigation to info screen.

Actual navigation parameters and pdf error handling would be covered in next PRs.
